src/graphics/program-lib/chunks/baseNineSliced.vert

```diff
@@ -0,0 +1,8 @@
+#define NINESLICED
+
+uniform vec4 innerOffset;
+uniform vec2 outerScale;
+uniform vec4 atlasRect;
```

Comment:
Change these three uniforms to be 'uniform mediump vec...'. This is because they're also declared in the vertex shader. If the device only supports mediump, they'll have highp in the vertex shader and mediump in the fragment shader and the program will not link. I don't believe there are any lowp devices out there so mediump should be fine.